src/components/auth/Oauth/Salesforce/SalesforceSubdomainEntry.tsx

```diff
@@ -1,59 +1,47 @@
-import { ExternalLinkIcon } from '@chakra-ui/icons';
-import {
-  Button, FormControl,
-  FormLabel, Heading, Input,
-} from '@chakra-ui/react';
-
 import { AuthErrorAlert } from 'components/auth/AuthErrorAlert/AuthErrorAlert';
+import { FormComponent } from 'components/form';
 import { AccessibleLink } from 'components/ui-base/AccessibleLink';
+import { Button } from 'components/ui-base/Button';
+import { isChakraRemoved } from 'src/components/ui-base/constant';
 import { AuthCardLayout } from 'src/layout/AuthCardLayout/AuthCardLayout';
 
-const SALESFORCE_HELP_URL = 'https://help.salesforce.com/s/articleView?id=sf.faq_domain_name_what.htm&type=5';
-
-type SubdomainEntryProps = {
-  handleSubmit: () => void;
-  setWorkspace: (workspace: string) => void;
-  error: string | null;
-  isButtonDisabled?: boolean;
-};
+import { ChakraSalesforceSubdomainEntry } from './ChakraSalesforceSubdomainEntry';
+import { SALESFORCE_HELP_URL, SubdomainEntryProps } from './SubdomainEntryProps';
 
-/**
- * Salesforce specific subdomain entry component, use workspace entry for other providers.
- * @param param0
- * @returns
- */
 export function SalesforceSubdomainEntry({
   handleSubmit, setWorkspace, error, isButtonDisabled,
 }: SubdomainEntryProps) {
+  const props = {
+    handleSubmit, setWorkspace, error, isButtonDisabled,
+  };
+  if (!isChakraRemoved) {
+    return <ChakraSalesforceSubdomainEntry {...props} />;
+  }
+
   return (
     <AuthCardLayout>
-      <FormControl>
-        <FormLabel>
-          <Heading as="h4" size="md">Enter your Salesforce subdomain</Heading>
-        </FormLabel>
-        <AccessibleLink href={SALESFORCE_HELP_URL} newTab>
-          {'What is my Salesforce subdomain? '}
-          <ExternalLinkIcon />
-        </AccessibleLink>
-        <AuthErrorAlert error={error} />
-        <div style={{ display: 'flex', marginTop: '1em' }}>
-          <Input
-            placeholder="MyDomain"
-            onChange={(event) => setWorkspace(event.currentTarget.value)}
-          />
-          <p style={{ lineHeight: '2.2em', marginLeft: '0.4em' }}>.my.salesforce.com</p>
-        </div>
-        <br />
-        <Button
-          variant="primary"
-          isDisabled={isButtonDisabled}
-          width="100%"
-          type="submit"
-          onClick={handleSubmit}
-        >
-          Next
-        </Button>
-      </FormControl>
+      <h1 style={{ fontWeight: 600, lineHeight: 1.2, fontSize: '1.2em' }}>Enter your Salesforce subdomain</h1>
+      <AccessibleLink href={SALESFORCE_HELP_URL} newTab>
+        What is my Salesforce subdomain?
+      </AccessibleLink>
+      <AuthErrorAlert error={error} />
+      <div style={{ display: 'flex', marginTop: '1em' }}>
+        <FormComponent.Input
+          id="salesforce-subdomain"
+          type="text"
+          placeholder="MyDomain"
```

Comment:
I would lowercase this because subdomains would generally look like `my-domain` or `mydomain`